tests/integration/README.md

```diff
@@ -29,18 +30,31 @@ case, just typing ```go test ./...``` should be sufficient to run tests.
 The test framework is designed to work with standard go tooling and allows developers
 to write environment-agnostics tests in a high-level fashion. The quickest way to get started with authoring
 new tests is to checkout the code in the
-[examples](https://github.com/istio/istio/tree/master/tests/integration/examples) folder.
+[framework](https://github.com/istio/istio/tree/master/tests/integration/framework) folder.
 
 The test framework has various flags that can be set to change its behavior.
 
- 
+#### Notes
+
+* Running on a Mac
+  * Currently some _native_ tests fail when being run on a Mac with an error like:
+    ```
+    unable to locate an Envoy binary
+    ```
+    This is documented in this [PR](https://github.com/istio/istio/issues/13677). Once the Envoy binary is available for the Mac,
+    these tests will hopefully succeed.
+  * If one uses Docker for Mac for the kubernetes environment be sure to specify the `-istio.test.kube.minikube` parameter. The solves an error like:
```

Comment:
not just d4m, anything that doesn't support LoadBalancer type

Reply:
Added comment to the command line flag as it isn't necessarily Mac specific.